build.gradle

```diff
@@ -414,3 +414,6 @@ task codeCoverageReport(type: JacocoReport, dependsOn: allprojects*.test) {
         xml.destination = file("${buildDir}/reports/jacoco/jacocoTestReport.xml")
     }
 }
+
+// for cnp pipeline
+task dependencyCheck(dependsOn: dependencyCheckAnalyze)
```

Comment:
Where is `dependencyCheckAnalyze`? Is it supposed to be another task?

Reply:
New pipeline should be dependencyCheckAnalyze
Old pipeline was dependencyCheck
And there can be more than one pipeline out there

Reply:
OK, that's fine